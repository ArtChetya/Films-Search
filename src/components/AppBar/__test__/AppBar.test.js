import { shallow } from 'enzyme/build';
import React from 'react';
import { AppBar } from '..';

describe('AppBar component', () => {
    it('should be rendered correctly', () => {
        const component = shallow(
            <AppBar>
                <div>Test</div>
            </AppBar>
        );
        expect(component).toMatchSnapshot();
    });
});
