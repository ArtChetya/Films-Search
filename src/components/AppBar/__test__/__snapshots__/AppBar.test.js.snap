// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppBar component should be rendered correctly 1`] = `ShallowWrapper {}`;
